app/component/map/tile-layer/CityBikes.js

```diff
@@ -3,7 +3,14 @@ import Protobuf from 'pbf';
 import Relay from 'react-relay';
 import glfun from 'mapbox-gl-function';
 import config from '../../../config';
-import { drawRoundIcon, getImageFromSprite } from '../../../util/mapIconUtils';
+import {
```

Comment:
Could there be a function called drawAvailabilityBadge, where you give the availability as a parameter?